modules/rostests/apitests/ntdll/NtQueryInformationProcess.c

```diff
@@ -3,12 +3,137 @@
  * LICENSE:         LGPLv2.1+ - See COPYING.LIB in the top level directory
  * PURPOSE:         Tests for the NtQueryInformationProcess API
  * PROGRAMMER:      Thomas Faber <thomas.faber@reactos.org>
+ *                  George Bișoc <george.bisoc@reactos.org>
  */
 
 #include "precomp.h"
 
 static LARGE_INTEGER TestStartTime;
 
+static
+void
+Test_ProcessBasicInformationClass(void)
```

Comment:
I thought you were going to test these using ps_i.h from https://github.com/reactos/reactos/pull/2874?